src/Builder/FormContractor.php

```diff
@@ -13,7 +13,7 @@
 
 namespace Sonata\DoctrineMongoDBAdminBundle\Builder;
 
-use Doctrine\ODM\MongoDB\Mapping\ClassMetadataInfo;
+use Doctrine\ODM\MongoDB\Mapping\ClassMetadata;
```

Comment:
This class exists in odm 1.x and 2.x. You could support both version inside the composer.json file